modules/shortcodes/js/jetpack-email-subscribe-block.js

```diff
@@ -0,0 +1,51 @@
+/* global wp */
+/* jshint esversion: 5, es3:false */
+const el = wp.element.createElement,
+	registerBlockType = wp.blocks.registerBlockType,
+	ServerSideRender = wp.components.ServerSideRender,
+	TextControl = wp.components.TextControl,
+	InspectorControls = wp.editor.InspectorControls;
+
+const fields = [
+	{ id: 'title', label: 'Title' },
+	{ id: 'email_placeholder', label: 'Placeholder' },
+	{ id: 'submit_label', label: 'Submit button label' },
+	{ id: 'consent_text', label: 'Consent text' },
+	{ id: 'processing_label', label: '"Processing" status message' },
+	{ id: 'success_label', label: 'Success status message' },
+	{ id: 'error_label', label: 'Error status message' },
+];
+
+registerBlockType( 'jetpack/email-subscribe', {
+	title: 'Email Subscribe Block',
```

Comment:
Should `title` be translatable? I would remove `Block` keyword from the title.

You can also add up to three keywords to make it easier to find.

I think Jetpack blocks use their own category, I think @tyxla worked on it.

Reply:
I added translations to everything that should be translatable, and dropped the "Block"